src/app/components/Persist/LoginModalLayout.tsx

```diff
@@ -1,26 +1,43 @@
 import { Box } from 'grommet/es6/components/Box'
 import { ResponsiveLayer } from '../ResponsiveLayer'
-import React from 'react'
+import React, { useContext } from 'react'
 import { Header } from 'app/components/Header'
 import { Navigation } from '../Sidebar'
+import walletBlueLogotype from '../../../../public/Rose Wallet Logo Blue cropped.svg'
+import walletWhiteLogotype from '../../../../public/Rose Wallet Logo White cropped.svg'
+import { ThemeContext } from 'styled-components'
 
 export function LoginModalLayout(props: {
   title: string
   children: React.ReactNode
   onClickOutside?: () => void
   onEsc?: () => void
 }) {
+  const { dark } = useContext<any>(ThemeContext)
+
   return (
     <Box direction="row-responsive" background="background-back" fill style={{ minHeight: '100dvh' }}>
       <Navigation />
       <ResponsiveLayer
         modal
+        responsive={false}
+        style={{
+          width: 'min(100%, 500px)',
+        }}
         background="background-front"
         onClickOutside={props.onClickOutside}
         onEsc={props.onEsc}
       >
         <Box pad="medium">
-          <Header level={2} textAlign="center" margin={{ top: 'medium' }}>
+          <Box align="center" margin={{ vertical: 'large' }}>
+            <img
+              alt="ROSE Wallet logo"
+              src={dark ? walletWhiteLogotype : walletBlueLogotype}
+              style={{ height: '65px' }}
```

Comment:
Avoid inline styles. Also, logo looks a bit large to me. Can we scale it down a bit?

Compared to logo in the navigation bar:
<img width="838" alt="image" src="https://github.com/oasisprotocol/oasis-wallet-web/assets/9722540/6bb25a3c-700a-42bb-9104-1daf27a0ce11">

Reply:
I was looking at old extension, but maybe that isn't intentionally that large: 
![image](https://github.com/oasisprotocol/oasis-wallet-web/assets/3758846/bc31f161-ba73-461d-9e69-92a680ec0c1a)

I'll reduce to 60 :shrug: